test/srt/test_double_sparsity.py

```diff
@@ -0,0 +1,64 @@
+import os
+import unittest
+from types import SimpleNamespace
+
+from sglang.srt.utils import kill_child_process
+from sglang.test.run_eval import run_eval
+from sglang.test.test_utils import (
+    DEFAULT_MODEL_NAME_FOR_TEST,
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    popen_launch_server,
+)
+
+
+class TestDoubleSparsity(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = DEFAULT_MODEL_NAME_FOR_TEST
+        cls.base_url = DEFAULT_URL_FOR_TEST
+        dirpath = os.path.dirname(__file__)
+        config_file = os.path.join(dirpath, "Llama-3.1-8B-Instruct.json")
+        cls.process = popen_launch_server(
+            cls.model,
+            cls.base_url,
+            timeout=DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+            other_args=[
+                "--enable-double-sparsity",
+                "--attention-backend",
+                "triton",
+                "--disable-cuda-graph",
```

Comment:
set them automatically here https://github.com/sgl-project/sglang/blob/869f1c02c4a7140c674ea92127a45eac0211bf74/python/sglang/srt/model_executor/model_runner.py#L94-L100, similar to the MLA optimization